app/assets/stylesheets/sephora_style_guide/partials/_product-card.scss

```diff
@@ -94,7 +94,7 @@
       left: 0;
       height: 30px;
       width: 20px;
-      background: asset-url('/assets/sephora_style_guide/left_arrow.png') center no-repeat;
```

Comment:
@ycewe Inside Rails CSS, the `asset-url` will automatically prefix the url provided with `/assets`, thus you should not include it because it actually doesn't work as intended. The fact that it may work on local is pure luck because the asset is named the same, not the case when you move to production where the asset is fingerprinted 😅